src/Logging/Logging.Testing/src/Microsoft.Extensions.Logging.Testing.csproj

```diff
@@ -0,0 +1,36 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
```

Comment:
Why isn't this in a test directory and non-packable?

Reply:
Put another way, this project doesn't need to be packaged unless dotnet/sdk or some such uses it

Reply:
Looks like in extensions, it was marked with IsShipping instead, I'll update it to use the convention we have in this repo. In fact, I think I'll move the source to M.A.Testing instead since there's no need for this to even be in a separate package.

Reply:
Sounds good